metadata-ingestion/sink_docs/datahub.md

```diff
@@ -0,0 +1,42 @@
+# DataHub
+
+## DataHub Rest
+
+To install this plugin, run `pip install 'acryl-datahub[datahub-rest]'`.
```

Comment:
It feels like we may want a few sections here:

1. Installation
2. Capabilities (or Description)
3. Configuration